libs/libc/wqueue/work_usrthread.c

```diff
@@ -193,15 +193,24 @@ static void work_process(FAR struct usr_wqueue_s *wqueue)
     {
       struct timespec rqtp;
       time_t sec;
+      uint64_t nsec;
```

Comment:
not all platform support uint64_t

Reply:
yes, thanks. Let's just write this in cleaner way, by using clock timespec functions already provided by sched. No point in duplicating the logic in many places.

Reply:
I see the code has already changed, but just for reference:  uint64_t is not available on all platforms supported by NuttX.  Its use should be guarded by CONFIG_HAVE_LONG_LONG which gets defined in nuttx/compiler.h